### ISSUE TITLE
Light mode/theme

### ISSUE THREAD

**Issue by CalAlaera** [JosefNemec/Playnite#3493]
### Feature description

Playnite does not have a light mode, which is needed for accessibility by users with certain vision problems. All of the available themes seem to be white-on-black. Ideally Playnite would switch between light or dark mode respecting the user's Windows settings, but even a manual toggle would be much appreciated. Thanks!

**Comment by Gerren** [comment]
Hi there. 

Your request seems very reasonable. But since Playnite is modular in the concept, there are themes for that, aren't they...?

For white on dark, see [StandardX](https://github.com/TheKersalMassive/StandardX)

As far as the light mode, there doesn't seem to be any. But you could ask some author, maybe they could make one...

**Comment by CalAlaera** [comment]
I did it myself.
